frontend.yaml

```diff
@@ -214,11 +274,11 @@
   isNew: FALSE
   description: Technical SEO - Schema.org, JSON-LD, OpenGraph, ...
 -
-  name: ARIA
+  name: A11Y
```

Comment:
Ik zou ARIA wel in de naam laten staan, dit is een belangrijk onderdeel van A11Y. 
A11Y/ARIA lijkt me prima